neuraxle/base.py

```diff
@@ -2029,6 +2356,27 @@ def __repr__(self):
         return output
 
 
+class MetaStep(MetaStepMixin, BaseStep):
+    def __init__(
+            self,
+            wrapped: BaseTransformer = None,
+            hyperparams: HyperparameterSamples = None,
+            hyperparams_space: HyperparameterSpace = None,
+            name: str = None,
+            savers: List[BaseSaver] = None,
+            hashers: List[BaseHasher] = None
+    ):
+        BaseStep.__init__(
+            self,
+            hyperparams=hyperparams,
+            hyperparams_space=hyperparams_space,
+            name=name,
+            savers=savers,
+            hashers=hashers
+        )
+        MetaStepMixin.__init__(self, wrapped=wrapped)
+
+
 class MetaStepJoblibStepSaver(JoblibStepSaver):
```

Comment:
Idea for another issue / another PR : the MetaStepJoblibSaver could be used for both metasteps and truncable steps using the new _HasCHildrens or _IsRecursive or something like that, using `get_childrens`.

Reply:
By looping on the get childrens for instance.  and by processing the stripped self after.